scripts/convert_hf_checkpoint.py

```diff
@@ -116,7 +116,10 @@ def permute(w, n_head):
     print(f"Saving checkpoint to {checkpoint_dir / 'model.pth'}")
     torch.save(final_result, checkpoint_dir / "model.pth")
     if 'llama-3' in model_name.lower():
-        original_dir = checkpoint_dir / "original"
+        if 'llama-3.1' in model_name.lower():
+            original_dir = checkpoint_dir / "original" / "mp16"
```

Comment:
what is this for?

Reply:
This is because llama-3.1 ```original``` folder has two sub folders: ```mp16``` and ```mp8```, we need to point to one of them to get ```tokenizer.model```. Previously, ```tokenizer.model``` was located at ```original``` folder directly.